backends/core/models.py

```diff
@@ -249,9 +316,48 @@ def destroy(self):
       Param.destroy(*param_ids)
     self.delete()
 
+  def get_status(self, max_retires=10):
+    key = str(self.pipeline_id) + '_' + str(self.id) + '_' + CACHE_KEY_STATUS
+    retries = 0
+    while retries < max_retires:
+      status = self.memcache_client.gets(key)
+      if status is None: 
+        '''The status is uninitialized in memcache, 
```

Comment:
For a multiline comment inside an implementation, it's more Pythonic to use `#`, like so:

```python
if status is None:
  # The status is uninitialized in memcache,
  # getting it from the database and
  # updating its value in memcache
  self.memcache_client.set(...)
```

Reply:
done